src/plots/plotGroup.ts

```diff
@@ -0,0 +1,33 @@
+module Plottable.Plots {
+  export class PlotGroup extends Components.Group {
+    public entityNearest(point: Point): PlotEntity {
+      let closestPlotEntity: PlotEntity;
+      let minDistSquared = Infinity;
+      this.components().forEach((plot: Plot) => {
+        let candidatePlotEntity = plot.entityNearest(point);
+        if (candidatePlotEntity == null) {
+            return;
+        }
+
+        let position = candidatePlotEntity.position;
+        let distSquared = Math.pow(position.x - point.x, 2) + Math.pow(position.y - point.y, 2);
```

Comment:
Looking at the existing plot.entityNearest definition, it looks like we only use the entity position, so this is fine. However, switch this to use the utility method: Utils.Math.distanceSquared